src/agent/main.cpp

```diff
@@ -66,6 +66,7 @@ enum
     OTBR_OPT_INTERFACE_NAME          = 'I',
     OTBR_OPT_VERBOSE                 = 'v',
     OTBR_OPT_VERSION                 = 'V',
+    OTBR_OPT_AUTO_ATTACH             = 'a',
```

Comment:
maybe we don't need a short cut, `--auto-attach` is good enough?

Reply:
Done.